python/lsst/dax/imgserv/cli.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+'''
+This module implements the command line interface (CLI) to ImageServ.
+
+@author: Brian Van Klaveren, SLAC
+@author: Kenny Lo, SLAC
+
+'''
+
+import os
+import json
+
+import click
+
+import lsst.log as log
+import lsst.daf.persistence as dafPersist
+
+from .locateImage import (
+        image_open, W13DeepCoaddDb, W13RawDb, W13CalexpDb
+)
+
+from .image import Image
+from .dispatch import Dispatcher
+from .hashutil import Hasher
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
```

Comment:
The intention was to use the pass-in config value.  If not passed in, then use the hardcoded default (as a guess perhaps, more complex discovery logic possible in the future) - code corrected to support the original intention.